### ISSUE TITLE
Homebridge and iOS11?

### ISSUE THREAD

**Issue by OliAhn** [nfarina/homebridge#1623]
Hi,

my homebridge with (home bridge-knx and Knxd) war working till iOS11. Since then is was temporarily broken, but with es reboot for that day OK. Even though a reboot it stop working. I did everything what was supposed to do (change password, delete persist, reconnect Home). After that I reinstalled everything and I was happy that it was working - for some days. I got really frustrated. My Home Central is an iPad and I do have the feeling that this is maybe the failure. Or that I connect some of my Family Members to Home. Does anybody else has such an experience? Regards, Michi

**Comment by raffoul** [comment]
Hi,

I've the same issue with an Apple TV. I had no problems with ios10.
I don't know how to resolve it. It stop working, I change my pin code / mac address, and it stops working again ...

**Comment by bencharlesrhys** [comment]
Did you do an npm update homebridge? This is all I needed to do to get it going on iOS 11, albeit with an Apple TV, not an iPad as the Central device.

**Comment by raffoul** [comment]
Yes. I tried with npm update then I tried to install it from scratch. Same issue a few days later.

**Comment by snizzleorg** [comment]
I have a similar issue. Although not sure if this is related to ios11. hombridge was working nicely but no stopped to find home bridge. home bridge works and receives updates (in my case from home-assistant) but there is no trace of communication with my iOS devices.

I re-added home bridge in iOS and it worked for a day. But now iOS cannot communicate with home bridge again.

**Comment by aswathraja** [comment]
I have been using on iOS 11.1.2 with a iPad Air 2 as my Home Hub. I have 2 6S Plus on iOS 11.1.2 and a 5S on iOS 10. It seems to be working just fine, has been working for 3 days now without a reboot. It does work to remotely control as well. Automation rules fail at sometimes but  only the rules which contain other accessories being controlled. The time of day or location based automation rules work flawlessly.
